### ISSUE TITLE
How to build QtSpeedTest?

### ISSUE THREAD

**Issue by shujaatak** [Aseman-Land/QtSpeedTest#1]
I have never build a Qt module before so can you please explain how to build QtSpeedTest?
When I build it in QtCreator in Windows 10 using MSVC 2015 it generates files and folders with makefile. I don't know what to do next :(

**Comment by shujaatak** [comment]
I am really in need to understand how to create and build custom modules in Qt :( 
Can you help me in this regard or point me to some helping material please!

**Comment by realbardia** [comment]
Hi
It's easy. First you need to install ```active perl``` in the windows.
In the windows using QtCreator open projects, and from side menu in the ```Projects > Build Steps``` click on the ```Add Build Step``` and add a Make step and in the ```Make Arguments``` field type ```install```.
It will build and install the module in the right place :)

**Comment by shujaatak** [comment]
The module built successfully, bin contains .dll files, lib contains .lib files etc but the include folder has only QtSpeedTestDepends file whereas it should have all required header files! Can you guess what I may be doing wrong? Would I need to manually place module header files in the include directory?

Note I have installed ActivePerl and added to path.

**Comment by shujaatak** [comment]
Ah after a long time I solved the issues like this:
Perl was not detected by Qt even though I was able to execute `perl -v` command in the Command Prompt, Very Strange!
To solve this issue I moved C:\Perl64\bin up and C:\Perl64\site\bin down in the Edit Environment Variable window.
Next even though Qt detected the perl but it was not generating the headers in the include directory so to resolve this issue I simply used CONFIG += git_build in .qmake.conf file.
Now on compilation everything worked as expected, all header files were generated in the include directory and proper .dll and .lib files were generated and installed in their proper locations.
By the way, I am much thankful to you for the steps you have mentioned above which helped me a lot!
